docs/Navigation/SubscribeWayPoints/assets/SubscribeWayPoints_Resumption.txt

```diff
@@ -0,0 +1,22 @@
+participant App
```

Comment:
this file as well as file SubscribeWayPoints_Resumption.png does not belong to SDL188